mlserver/cli/constants.py

```diff
@@ -1,12 +1,14 @@
 DockerfileName = "Dockerfile"
 DockerfileTemplate = """
-FROM continuumio/miniconda3:4.12.0 AS env-builder
+FROM continuumio/miniconda3:22.11.1 AS env-builder
```

Comment:
I just noticed that the main `Dockerfile` actually uses `23.1.0`, so should we use that instead to make sure they're aligned?

Reply:
I wasn't able to test with that version of the docker image on a linux host machine and got some awful c++ issues locally with our custom runtimes (might be me trying to test on a Mac host but not entirely sure) ... still want me to bump to highest conda image?

Reply:
Fair enough - in that case let's go ahead with the PR as-is :+1:

Reply:
I would add one thing to consider for the future here ... image CVEs are becoming a big part of our work so having both build and inference image at the highest python and even debian as well as library versions is becoming something we focus on. I can see you guys being super proactive on lib CVEs via dependabot but this is a level below that to consider also.